test/tomo/operators/spect_trafo_test.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for the Single-photon emission computerized tomography transform."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+# External
+import pytest
+import numpy as np
+
+# Internal
+import odl
+from odl.tomo.operators.spect_trafo import AttenuatedRayTransform
+from odl.util.testutils import skip_if_no_niftyrec
+from odl.util.testutils import almost_equal, all_equal
+
+
+@skip_if_no_niftyrec
+def test_spect_projector():
+    """Test discrete SPECT transform."""
+
+    # Geometry
+    det_nx_pix = 32
+    det_ny_pix = 32
+    det_nx_mm = 4
+    det_radius = 200
+    n_proj = 16
+    det_param = det_nx_mm * det_nx_pix
+    dpart = odl.uniform_partition([-det_param, -det_param],
+                                  [det_param, det_param],
+                                  [det_nx_pix, det_ny_pix])
+
+    apart = odl.uniform_partition(0, 2 * np.pi, n_proj)
+    geometry = odl.tomo.geometry.spect.ParallelHoleCollimatorGeometry(
+        apart, dpart, det_rad=det_radius)
```

Comment:
In [spect geometry](https://github.com/odlgroup/odl/blob/issue-387__spect/odl/tomo/geometry/spect.py#L71). But not anywhere else, since unfortunately `NiftyRec` does not support contoured orbits..